lib/nostrum/struct/sticker.ex

```diff
@@ -0,0 +1,164 @@
+defmodule Nostrum.Struct.Sticker do
+  @moduledoc """
+  A `Nostrum.Struct.Sticker` represents a sticker that can be sent inside a
+  `Nostrum.Struct.Message` and created/modified with the relevant API methods
+  """
+
+  alias Nostrum.Struct.{Guild, User}
+  alias Nostrum.{Constants, Snowflake, Util}
+
+  defstruct [
+    :id,
+    :pack_id,
+    :name,
+    :description,
+    :tags,
+    :type,
+    :format_type,
+    :available,
+    :guild_id,
+    :user,
+    :sort_value
+  ]
+
+  @typedoc """
+  ID of the sticker
+  """
+  @type id :: Snowflake.t()
+
+  @typedoc """
+  ID of the pack the sticker is from
+  """
+  @type pack_id :: Snowflake.t()
+
+  @typedoc """
+  Name of the sticker
+  """
+  @type name :: String.t() | nil
+
+  @typedoc """
+  Description of the sticker
+  """
+  @type description :: String.t() | nil
+
+  @typedoc """
+  Tags used by the Discord client to auto-complete a sticker.
+
+  For default sticker packs, this is a comma-separated list. For guild stickers,
+  this is the name of the unicode emoji associated by the sticker creator with
+  the sticker.
+
+  This is technically a free-text field so consistency in formatting is not guaranteed.
+  """
+  @type tags :: String.t()
+
+  @typedoc """
+  Type of the sticker as an integer enum.
+  """
+  @type type :: :standard | :guild
```

Comment:
This says it's an integer enum but then we spec it as atoms. Shouldn't we just use "Type of the sticker" and explain that a standard sticker is a global thing and guild a guild-specific one?